src/main/java/az/edu/turing/Console.java

```diff
@@ -0,0 +1,228 @@
+                                adoptBaby();
+                                break;
+                            case "3":
+                                break label2;
+                            default:
+                                System.out.println("Invalid command");
+                        }
+                    }
+                    break;
+                case "9":
+                    System.out.println("Enter the age: ");
+                    int age = getValidNumber();
+                    familyController.getAllFamilies().forEach(family -> familyController.deleteAllChildrenOlderThen(age));
+                    break;
+                default:
+                    System.out.println("Invalid command");
+            }
+        }
+        sc.close();
+    }
+
+    private int getValidNumber() {
+        while (true){
+            if (sc.hasNextInt()) {
+                return sc.nextInt();
+            }
+            sc.nextLine();
+            System.out.print("Input must be integer!!! Try again: ");
+        }
+    }
+
+    private void adoptBaby() {
+        int familyId = getFamilyId();
+
+        System.out.println("Enter the full name of the child: ");
+        String fullName = sc.nextLine();
+        String[] names = fullName.split(" ");
+        String name = names[0];
+        String lastName = names[1];
+        String birthDate = getBirthDate("child");
+        int iq = getIq("child");
+
+        Human adoptedChild = new Human(name, lastName, birthDate, iq);
+
+        familyController.adoptChild(familyController.getFamilyById(familyId), adoptedChild);
+    }
+
+
+    private void birthBaby() {
+        int familyId = getFamilyId();
+
+        System.out.println("If it is boy, what will be his name?");
+        String boyName = sc.nextLine();
+        System.out.println("If it is girl, what will be her name?");
+        String girlName = sc.nextLine();
+
+        Family family = familyController.getFamilyById(familyId);
+        familyController.bornChild(family, boyName, girlName);
+    }
+
+    private int getFamilyId() {
+        int familyId;
+        while (true) {
+            System.out.print("Please enter family id: ");
+            familyId = getValidNumber();
+            System.out.println(familyController.count());
+            if (familyId < 0 || familyId >= familyController.count()) {
+                System.out.println("Invalid family id!");
+                continue;
+            }
+            break;
+        }
+        return familyId;
+    }
+
+    private void createNewFamily() {
+        System.out.println("Let's create a new family.");
+
+        Human mother = requestMemberFields("mother");
+        Human father = requestMemberFields("father");
+
+        Family family = new Family(mother, father);
+
+        mother.setFamily(family);
+        father.setFamily(family);
+
+        familyDao.saveFamily(family);
+    }
+
+    private Human requestMemberFields(String member) {
```

Comment:
you can reuse this method for adoptChild also, instead of sending member field, you can send boolean value isMale or isFemale